packages/core/src/modules/connections/ConnectionsApi.ts

```diff
@@ -227,6 +232,37 @@ export class ConnectionsApi {
     return connectionRecord
   }
 
+  /**
+   * Send a trust ping to an established connection
+   *
+   * @param connectionId the id of the connection for which to accept the response
+   * @param responseRequested do we want a response to our ping
+   * @param withReturnRouting do we want a response at the time of posting
+   * @returns boolean indicating success or failure
```

Comment:
This method doesn't return a boolean anymore

Reply:
Fixed